<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="eli-mpb-4.local" name="org.artifactory.util.ExceptionUtilsTest" tests="4" failures="0" timestamp="27 Nov 2014 18:02:53 GMT" time="0.001" errors="0">
  <testcase name="testGetRootCauseNotNested" time="0.000" classname="org.artifactory.util.ExceptionUtilsTest"/>
  <testcase name="testGetCauseOfTypes" time="0.001" classname="org.artifactory.util.ExceptionUtilsTest"/>
  <testcase name="testUnwrapThrowablesOfTypes" time="0.000" classname="org.artifactory.util.ExceptionUtilsTest"/>
  <testcase name="testGetRootCauseNested" time="0.000" classname="org.artifactory.util.ExceptionUtilsTest"/>
</testsuite> <!-- org.artifactory.util.ExceptionUtilsTest -->
